.github/workflows/delete_diffs.yaml

```diff
@@ -0,0 +1,27 @@
+name: Delete Installation Diffs
+
+on:
+  workflow_call:
+
+jobs:
+  # If comment with `delete_diffs` has been added the job gets all the installations, and feeds them
+  # to the next job that removes the installations diffs from the PR.
```

Comment:
```suggestion
  # If a comment with the keyword `delete_diffs` is added to the PR,
  # the job gets all the installations and feeds them to the next job
  # that removes the installations diffs from the PR.
```

Reply:
Above it says the command is `delete_diffs`. In line 11 it looks like the command needed is

    /deletediffs